### PR TITLE
anki: Add appstream metainfo

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Add app icon
- Add appstream metainfo (Part of getsolus/packages#1389)

**Test Plan**

Install and run the app

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/a/anki/files/net.ankiweb.Anki.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <image type="source">https://iili.io/HU8RpwX.png</image>
+      <caption>Flashcard and answer</caption>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1"/>
+  <provides>
+    <id>net.ankiweb.Anki.desktop</id>
+  </provides>
+  <launchable type="desktop-id">anki.desktop</launchable>
+  <update_contact>dan_at_reactivated.net</update_contact>
```
Comment (EbonJaeger) [inline packages/a/anki/files/net.ankiweb.Anki.appdata.xml:35]: Should this be included? I'd hate to have that shown to people when they should be coming to us to ask for updates. Unfortunately, I can't find any documentation on what that key is supposed to be for.

**Review by EbonJaeger — COMMENTED** [review]
Just one question. Looks good, otherwise.

File: packages/a/anki/files/net.ankiweb.Anki.appdata.xml (hunk 1 above)
Comment (malfisya) [reply packages/a/anki/files/net.ankiweb.Anki.appdata.xml:35]: Hmmm, I can remove that. It doesn't affect validation. 
It barely shows in GS, It should be grouped together with the other URL in middle right of app page, just above comments.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**